content/en/BestPractices/prevent-security-misconfig.md

```diff
@@ -0,0 +1,160 @@
+          </system.web>
+          ```
+
+### Directory Listing
+
+Directory listing allows users to see all files and directories stored on a web server. A malicious actor may get unauthorized access to sensitive files. Search the web with `intitle: "index of /"` to see websites that have directory listing enabled.
+
+In the following example, we show how to prevent this problem for an Apache Tomcat web server:
+
+1. Open the `conf/web.xml` file in the directory where Tomcat is installed.
+
+     - In Windows 10, the default installation directory is `C:\Program Files (x86)\Apache Software Foundation\Tomcat 9.0`.
+
+1. In the file, set `<param-value>false</param-value>` to `false` for `<param-name>listings</param-name>`:
+
+     ```
+     <servlet>
+          <servlet-name>default</servlet-name>
+          <servlet-class>org.apache.catalina.servlets.DefaultServlet</servlet-class>
+          <init-param>
+               <param-name>debug</param-name>
+               <param-value>0</param-value>
+          </init-param>
+          <init-param>
+               <param-name>listings</param-name>
+               <param-value>false</param-value>
+          </init-param>
+          <load-on-startup>1</load-on-startup>
+     </servlet>
+     ```
+
+1. In the Apache web server, create a `.htaccess` file in the related directory.
+
+1. Add the following lines to the `httpd.conf` file, or replace existing lines with:
+
+     ```
+     <Directory /{YOUR DIRECTORY}>
+     Options FollowSymLinks
+     </Directory>
+     ```
+
+To turn directory listing off, modify the configuration files of your web server.
+
+### Default Accounts
+
+A developer may forget to remove default accounts when releasing the application.
+
+Examples of default accounts:
+
+- Application accounts
+- Web server logins
+- Cloud instances
+- Databases
+
+To prevent this:
+
+- Delete default accounts from the development environment once the development is complete
+- Change default passwords that come with installation packages
+- Restrict the exposure of administrator pages and web server logins in the production environment
+- Establish a strong password policy for your accounts
+
+### Files or Directories in a URL Path
+
+Sensitive files that may appear in a URL path include:
+
+- Backup files
+- Log files
+- Database files
+
+An attacker may discover sensitive files by brute-forcing the URL paths.
+
+To prevent this:
+
+- Remove any sensitive files or directories from the URL paths
+- Check your code before releasing it
+- To prevent URL fuzzing or a brute-force attack, apply a web application firewall and rate limiting checks
+
+### Misconfigured Cloud Services
+
+Misconfigurations in the cloud infrastructure or services may lead to data compromise. For example, an [Amazon S3 bucket](https://aws.amazon.com/s3/) with insufficient configuration may grant read and write access to an attacker.
+
+To prevent this:
+
+- Implement security automation to identify misconfigurations
```

Comment:
What's an example of "security automation" that someone could use to identify a misconfiguration? If you know of one, we could say:

```suggestion
- Implement security automation to identify misconfigurations, such as [something](with a link).
```

Reply:
Implement security automation to identify misconfigurations, such as using DAST tools in CI-CD pipelines to identify such vulnerabilities. Ref: https://www.zaproxy.org/alerttags/owasp_2021_a05/

Reply:
@shadixit I've modified your suggestion a bit:

- I've pointed directly to the zaproxy.org website (instead of the list of checks)
- I've removed the CI-CD reference, as people can use OWASP Zap from a UI (and I'm guessing also from the command line)